app/views/applications/process/confirmation.html.slim

```diff
@@ -3,6 +3,8 @@ header
     .small-12.medium-12.large-12.columns
       h2 Processing complete
 
+= display_reference(@application)
```

Comment:
This will have no formatting, will it? Should we have the same visual style as on the evidence check letter when application is selected for evidence?